src/Web/ClientApp/src/app/components/application/new/new.component.html

```diff
@@ -59,6 +59,9 @@
 				<div *ngIf="submitted && f['storageId'].errors" class="panel-block help is-danger">
 					<div *ngIf="f['storageId'].errors['required']">Storage ID is required.</div>
 				</div>
+				<div *ngIf="!storageList?.length" class="panel-block help is-danger">
+					No bindles available. <span *ngIf="!storageQuery.length">&nbsp;Add by running `spin bindle push --file &lt;spin_file&gt;` </span>
```

Comment:
I think we can omit the `--file` flag as that is set to spin.toml by default: https://github.com/fermyon/spin/blob/b4ca6d15c300a4382d5fc6cc2211c3665f0bfd44/src/commands/bindle.rs#L122-L125